### PR TITLE
Домашне_завдання_07

### PR DESCRIPTION
**Pull request by Hronologiya**

### CODE REVIEW

File: lesson_07/homework_07.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while multiplier <= 5:
+        result = number * multiplier
+        # десь тут помила, а може не одна
+        if  result > 25:
+            # Enter the action to take if the result is greater than 25
+            pass
```
Comment (DartBrovsky) [inline lesson_07/homework_07.py]: You need to take an action here. For ex. break or raise any exception. With pass keyword you only ignore this block of code and if check has no any sense.

File: lesson_07/homework_07.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    return s[::-1]
+input_string = "Привіт Світ"
+reversed_string = reverse_string(input_string)
+print("Зворотний рядок:", reversed_string)
+
+# task 5
+"""  Написати функцію, яка приймає список слів та повертає найдовше слово у списку.
+"""
+def longest_word(words):
+    if not words:
```
Comment (DartBrovsky) [inline lesson_07/homework_07.py:70]: You solution is ok. But we have max function that can be customizable and so on.

**Review by DartBrovsky — APPROVED** [review]
Your solution is okay. Only thing - please take a look into my comments and fix is you want. But I approved this solution any way.

File: lesson_07/homework_07.py (hunk 1 above)
Comment (Hronologiya) [reply lesson_07/homework_07.py]: Thank you for your advice, I fixed it.

**Review by DartBrovsky — APPROVED** [review]

**Pull request merged**